aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -899,17 +899,22 @@ Tensor cudnn_convolution_forward(
   checkAllSameType(c, {input, weight});
   checkAllSameGPU(c, {input, weight});
 
+  // Check for contiguous to avoid the ambiguity in `suggest_memory_format()`,
+  // Otherwise, it causes cudnn into unsupported configures with test failing
   auto output_t = at::empty(
                     conv_output_size(input->sizes(), weight->sizes(),
                                      padding, stride, dilation, groups),
-                    input->options());
+                    input->options(),
+                    input->is_contiguous() ? at::MemoryFormat::Contiguous : input->suggest_memory_format());
```

Comment:
Is it because sizeC == 1 Tensors considered channels last? How about we change input->suggest_memory_format() to prioritize `at::MemoryFormat::Contiguous` if tensor is_contiguous?

Reply:
Answered above.